src/components/views/messages/MLocationBody.tsx

```diff
@@ -144,13 +144,15 @@ export const LocationBodyContent: React.FC<LocationBodyContentProps> = ({
     return <div className="mx_MLocationBody">
         {
             tooltip
-                ? <TooltipTarget
+                ? <div className="mx_MLocationBody_tooltip">
+                    <TooltipTarget
                     label={tooltip}
                     alignment={Alignment.InnerBottom}
                     maxParentWidth={450}
                 >
                     { mapElement }
                 </TooltipTarget>
```

Comment:
The indentation here looks wrong. You need one more level of indentation in lines 149-154

Reply:
I have corrected the indentation